tests/test_api/test_submit_job.py

```diff
@@ -263,6 +263,9 @@ def test_submit_good_autorift_granule_names(client, tables):
         'S2A_1UCR_20210124_0_L1C',
         'S2B_22WEB_20200612_0_L1C',
         'S2A_22XEQ_20190610_10_L1C',
+        'LM04_L1GS_025009_19830519_20200903_02_T2',
+        'LT05_L1TP_091090_20060929_20200831_02_T1',
+        'LE07_L2SP_233095_20200102_20200822_02_T2',
```

Comment:
This might be more an issue for hyp3-autorift than hyp3, but this looks like a Level 2 scene, where everything else we've dealt with is Level 1. @jhkennedy should we restrict Landsat scene names to Level 1 scenes by requiring `L1` in the 6-7th characters?

hyp3-autorift explicitly uses the L1 URL when lookup up scenes in the Landsat STAC catalog: https://github.com/ASFHyP3/hyp3-autorift/blob/develop/hyp3_autorift/process.py#L37

This is a valid L2 scene and exists in S3 at `s3://usgs-landsat/collection02/level-2/standard/etm/2020/233/095/LE07_L2SP_233095_20200102_20200822_02_T2/`, but looking it up at https://landsatlook.usgs.gov/stac-server/collections/landsat-c2l1/items/LE07_L2SP_233095_20200102_20200822_02_T2 gives "not found".

The corresponding L1 scene is at `s3://usgs-landsat/collection02/level-1/standard/etm/2020/233/095/LE07_L1GT_233095_20200102_20200822_02_T2/`, and querying https://landsatlook.usgs.gov/stac-server/collections/landsat-c2l1/items/LE07_L1GT_233095_20200102_20200822_02_T2 finds it as expected.

Reply:
Yes, we'll typically be processing L1 scenes as the L2 adds in some corrections that mess with the feature tracking, so I think it'd be worth restricting to Level 1 scenes. 


For reference, here are the L7 test pairs we've been using to develop the wallis filter:
```
LE07_L1TP_063018_20040911_20200915_02_T1,LE07_L1TP_063018_20040810_20200915_02_T1
LE07_L1TP_061018_20050324_20200914_02_T1,LE07_L1TP_061018_20040321_20200915_02_T1
LE07_L1TP_061018_20040625_20200915_02_T1,LE07_L1TP_061018_20030810_20200916_02_T1
LE07_L1TP_061018_20040711_20200915_02_T1,LE07_L1TP_061018_20040625_20200915_02_T1
LE07_L1TP_061018_20050511_20200914_02_T1,LE07_L1TP_061018_20040625_20200915_02_T1
LE07_L1TP_061018_20040711_20200915_02_T1,LE07_L1TP_061018_20030810_20200916_02_T1
LE07_L1TP_061018_20040625_20200915_02_T1,LE07_L1TP_061018_20040321_20200915_02_T1
LE07_L1TP_061018_20050425_20200914_02_T1,LE07_L1TP_061018_20040321_20200915_02_T1
LE07_L1TP_062018_20050416_20200914_02_T1,LE07_L1TP_062018_20040803_20200915_02_T1
LE07_L1TP_063018_20050509_20200914_02_T1,LE07_L1TP_063018_20040810_20200915_02_T1
LE07_L1TP_062018_20040803_20200915_02_T1,LE07_L1TP_062018_20040718_20200915_02_T1
```

Reply:
I looked through the ITS_LIVE Landsat scene name bucket (s3://its-live-data/NSIDC/v01/velocity_image_pair) and confirmed that there are just L1 data there. I added a pattern to the check in job-spec/AUTRIFT*.yml and a failure case in `test_submit_bad_autorift_granule_names` (L305 in tests/test_api/test_submit_job.py).